CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptTs.c

```diff
@@ -374,8 +373,8 @@ ImageTimestampVerify (
   //
   if (SigningTime != NULL) {
     SetMem (SigningTime, sizeof (EFI_TIME), 0);
-    Status = ConvertAsn1TimeToEfiTime (Ptr, SigningTime);
+    return ConvertAsn1TimeToEfiTime (Ptr, SigningTime);
   }
 
-  return Status;
+  return FALSE;

```

Comment:
Shouldn't it return TRUE here?

Reply:
Good question.

The OUT parameter SigningTime of BaseCryptLib's ImageTimestampVerify() is not marked as OPTIONAL, and all callers in the tree appear to provide a non-NULL value here, so in practice, it doesn't make a difference.

But I'd argue that not providing SigningTime violates the API, and so FALSE should be returned. But perhaps that should be done much earlier (instead of the first SetMem())

Reply:
From function perspective, I would recommend making MbedTLS version be consistent with OpenSSL version.

I checked the openssl version (https://github.com/tianocore/edk2/blob/master/CryptoPkg/Library/BaseCryptLib/Pk/CryptTs.c#L512). It seems return TRUE, if SigningTime is NULL.

If the comment is confusing, we can fix the comment.

Reply:
OK I will return TRUE instead